src/main/kotlin/com/vk/kphpstorm/inspections/quickfixes/ReplaceToKphpFunctionsQuickFix.kt

```diff
@@ -0,0 +1,34 @@
+package com.vk.kphpstorm.inspections.quickfixes
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.jetbrains.php.lang.lexer.PhpTokenTypes
+import com.jetbrains.php.lang.psi.PhpPsiElementFactory
+import com.jetbrains.php.lang.psi.elements.FunctionReference
+
+class ReplaceToKphpFunctionsQuickFix(private val maybeFunctionName: String) : LocalQuickFix {
+    override fun getFamilyName(): String {
+        return "Replace with $maybeFunctionName()"
+    }
+
+    override fun applyFix(project: Project, descriptor: ProblemDescriptor) {
+        val unsupportedFunction = descriptor.psiElement
+        if (unsupportedFunction !is FunctionReference) {
```

Comment:
Here you use curly braces for `if` with a single return body, but above you don't. 

Judging by the rest of the project, you usually don't need to use curly braces at all for such `if`s.

Reply:
Fixed :)